WcaOnRails/lib/results_validators/persons_validator.rb

```diff
@@ -102,6 +107,20 @@ def validate(competition_ids: [], model: Result, results: nil)
                                            WRONG_PARENTHESIS_FORMAT_ERROR,
                                            name: p.name)
           end
+          if /[（）]/ =~ p.name
+            @errors << ValidationError.new(:persons, competition_id,
+                                           WRONG_PARENTHESIS_TYPE_ERROR,
+                                           name: p.name)
+          end
+          # Look for if 2 new competitors that share the exact same name
+          if without_wca_id.select{ |p2| p2.name == p.name }.length > 1 && !duplicate_newcomer_names.include?(p.name)
```

Comment:
RuboCop: add space before `{`

Reply:
Thanks! I was trying to figure out why the build failed. Forgot the RuboCop errors were at the top.